tox_docker.py

```diff
@@ -127,6 +129,19 @@ def _validate_port(port_line):
     return (host_port, container_port_proto)
 
 
+def _validate_link(envconfig, link_line):
+    name, _, alias = link_line.partition(":")
```

Comment:
just confirming: docker does the right thing if the `links` mapping (built on L210-213) has empty string as the value? It doesn't have to be eg `links["memcached"] = "memcached"`?

Reply:
Do you mean if the image is configured like:
```
[docker:postgres:alpine]
links = memcached:
```

?

Reply:
Can probably check the sep component from partition and output a misconfiguration warning when sep is detected without an alias?

Reply:
Either way:

```
>>> "memcached".partition(":")
('memcached', '', '')
>>> "memcached:".partition(":")
('memcached', ':', '')```

I _assume_ that it works, but since postgres doesn't actually try to talk to memcached (that I know of?) I'm not sure anything would fail if that link didn't get created appropriately.

Reply:
The docker alias is optional.  So either way is fine.  

Docker uses the container id if you don't specify an alias as demonstrated here: https://github.com/tox-dev/tox-docker/pull/58/files#diff-128d1bcc03a97318e597170b8a0a6fe5R42

I think it makes sense to warn the user if we catch a dangling colon because they probably meant to provide an alias.  I will update the other PR branch with the fix

Reply:
Resolved per https://github.com/tox-dev/tox-docker/pull/58#issuecomment-637578945